app/main.py

```diff
@@ -1,8 +1,27 @@
+from typing import List
+
+
 class Person:
-    # write your code here
-    pass
 
+    people: dict = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+
+def create_person_list(people: list) -> List[Person]:
+    result = [Person(name=data.get("name"),
+                     age=data.get("age")) for data in people]
+
+    for person in people:
+        if person.get("wife") is not None:
+            Person.people[person["name"]].wife \
+                = Person.people[person["wife"]]
```

Comment:
use `()` instead of `()`

Reply:
I don't understand what exactly to replace, if you can elaborate.